src/main/java/org/sagebionetworks/bridge/sdk/BridgeResearcherClient.java

```diff
@@ -258,15 +258,15 @@ public void deleteUploadSchemaByIdAndRevision(String schemaId, int revision) {
     }
 
     @Override
-    public UploadSchema getUploadSchemaById(String schemaId) {
+    public UploadSchema getUploadSchema(String schemaId) {
         session.checkSignedIn();
         checkArgument(isNotBlank(schemaId), Bridge.CANNOT_BE_BLANK, "schemaId");
         return get(config.getUploadSchemaByIdApi(schemaId), UploadSchema.class);
     }
 
     @Override
-    public ResourceList<UploadSchema> getUploadSchemaForStudy() {
+    public ResourceList<UploadSchema> getUploadSchemasAllRevisions() {
```

Comment:
I guess I think there logically could be two methods (even if only one is needed): one method that gets the latest revision of every upload schema, and one that gets all revisions of all schemas. So:

getAllUploadSchemas() <-- get head revision of all schemas
getAllUploadSchemasAllRevisions() <-- get all revisions of all schemas